pkg/minishift/registration/redhat_test.go

```diff
@@ -27,22 +27,18 @@ import (
 
 var (
 	param = &RegistrationParameters{
-		Username:        "foo",
-		Password:        "foo",
```

Comment:
> Actually, I don't think we should remove this. We should keep this in place. The whole idea is/was to allow other registrars.

We are not removing username/password it's just when I deleted proxy param it shifted bit down line.